From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Mon, 7 Apr 2014 17:00:15 +0100
Subject: doxyfile path fix

---
 CMake/GenerateDoxyfile.cmake | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/CMake/GenerateDoxyfile.cmake b/CMake/GenerateDoxyfile.cmake
index 4b00958..da37beb 100644
--- a/CMake/GenerateDoxyfile.cmake
+++ b/CMake/GenerateDoxyfile.cmake
@@ -25,5 +25,12 @@ string(REPLACE
     DOXYFILE_AUXVAR "${DOXYFILE_CONTENTS}"
 )
 
+# Change the STRIP_FROM_PATH so that it works right even in the build directory;
+# otherwise, every file will have the full path in it.
+string(REGEX REPLACE
+    "(STRIP_FROM_PATH[ ]*=) ./"
+    "\\1 ${CMAKE_CURRENT_SOURCE_DIR}/"
+    DOXYFILE_CONTENTS ${DOXYFILE_AUXVAR})
+
 # Save the Doxyfile to its new location.
-file(WRITE "${DESTDIR}/Doxyfile" "${DOXYFILE_AUXVAR}")
+file(WRITE "${DESTDIR}/Doxyfile" "${DOXYFILE_CONTENTS}")
