--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,9 +2,10 @@
 project(mlpack C CXX)
 
 # First, define all the compilation options.
-# We default to debugging mode for developers.
-option(DEBUG "Compile with debugging information" ON)
-option(PROFILE "Compile with profiling information" ON)
+# Default to clean for packaging and release; developers can call for debugging
+# and profiling if they need to.
+option(DEBUG "Compile with debugging information" OFF)
+option(PROFILE "Compile with profiling information" OFF)
 option(ARMA_EXTRA_DEBUG "Compile with extra Armadillo debugging symbols." OFF)
 option(MATLAB_BINDINGS "Compile MATLAB bindings if MATLAB is found." OFF)
 
