From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Mon, 7 Apr 2014 16:59:22 +0100
Subject: troff single quote

---
 CMake/exec2man.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMake/exec2man.sh b/CMake/exec2man.sh
index ce59e5b..b0807fc 100755
--- a/CMake/exec2man.sh
+++ b/CMake/exec2man.sh
@@ -64,7 +64,7 @@ synopsis="$name [-h] [-v] $reqoptions $options";
 # correctly so that the entire description of the parameter is on one line (this
 # helps avoid 'man' warnings).
 # The sed line at the end removes accidental macros from the output, replacing
-# 'word' with "word".
+# single-quotes at the beginning of a line with the troff escape code \(aq.
 ./"$name" -h | \
   sed 's/^For further information/Additional Information\n\n For further information/' | \
   sed 's/^consult the documentation/ consult the documentation/' | \
@@ -76,5 +76,5 @@ synopsis="$name [-h] [-v] $reqoptions $options";
   awk '/NAME/,/REQUIRED OPTIONS/ { print; } /ADDITIONAL INFORMATION/,0 { print; } /REQUIRED OPTIONS/,/ADDITIONAL INFORMATION/ { if (!/REQUIRED_OPTIONS/ && !/OPTIONS/ && !/ADDITIONAL INFORMATION/) { if (/ --/) { printf "\n" } sub(/^[ ]*/, ""); sub(/ [ ]*/, " "); printf "%s ", $0; } else { if (!/REQUIRED OPTIONS/ && !/ADDITIONAL INFORMATION/) { print "\n"$0; } } }' | \
   sed 's/  ADDITIONAL INFORMATION/\n\nADDITIONAL INFORMATION/' | \
   txt2man -P mlpack -t "$name" -d 1 | \
-  sed "s/^'\([A-Za-z0-9 ]*\)'/\"\1\"/" > "$output"
+  sed "s/^'/\\\\(aq/" > "$output"
 
